src/init.cpp

```diff
@@ -977,8 +988,7 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
     // Option to startup with mocktime set (used for regression testing):
     SetMockTime(GetArg("-mocktime", 0)); // SetMockTime(0) is a no-op
 
-    ServiceFlags nLocalServices = NODE_NETWORK;
-    ServiceFlags nRelevantServices = NODE_NETWORK;
+    nLocalServices = NODE_NETWORK;
```

Comment:
Any reason the default value was not assigned in the `namespace` (like `nRelevantServices`)?